build.gradle

```diff
@@ -1,5 +1,6 @@
 plugins {
 	id 'java'
+	id 'org.jetbrains.kotlin.jvm' version '1.7.0'
```

Comment:
I think the Kotlin stuff should be moved to a separate sub-project if it has to be a part of the master branch, just to avoid any potential issues with building on the plugin hub. There are no other hub plugins using Kotlin, even just for tests.

Maybe it would be okay just to run this once, or move it to a separate repo?

Reply:
It only needs to be ran once but since there is so many prs with environmental changes why its kept in, as soon as its merged I can make the new File and delete this

Reply:
I think it's better to keep it separate then, just to avoid the hassle with plugin hub builds. Feel free to keep it in the PR till we're ready to merge though :+1: